website/source/docs/agent/services.html.md

```diff
@@ -66,6 +66,19 @@ example shows all possible fields, but note that only a few are required.
       "upstreams": [],
       "mesh_gateway": {
         "mode": "local"
+      },
```

Comment:
It looks like many of the other stanzas in this service definition are explained in the paragraphs below it. We might want to give a wordier explanation of the expose stanza there and link out to the other places you can set it? This seems to be the place where we explain the most about service registrations and what can go into them.

Reply:
I noticed that there wasn't a reference for the proxy stanza in general, so instead of repeating all the content here I gave a brief description and linked out to the full proxy registration docs.